pkg/integrations/v2/app_o11y_receiver/models/trace.go

```diff
@@ -0,0 +1,83 @@
+package models
```

Comment:
I wouldn't add tracing just yet as they will not be part of the first draft

Reply:
My initial idea was to define accepted payload first, then implement various services using hte payload as a reference.
But as discussed, I'll comment out the traces member for now

Reply:
removed tracing for now